store/tikv/txn.go

```diff
@@ -138,10 +138,10 @@ func (txn *tikvTxn) Commit() error {
 	}
 	err = committer.Commit()
 	if err != nil {
-		committer.writeFinisheBinlog(binlog.BinlogType_Rollback, 0)
```

Comment:
don't we have a postDDL type? If so, why just leave three types `BinlogType_Prewrite, BinlogType_Rollback, BinlogType_Commit`